corehq/apps/dashboard/templates/dashboard/base.html

```diff
@@ -206,5 +206,9 @@ <h1>
       </p>
     </div>
   {% endif %}
+
```

Comment:
was there a specific reason to include the script tag on this template? `dashboard/base.html` ultimately inherits from `hqwebapp/base.html`

Reply:
No specific reason I can think of right now.